### ISSUE TITLE
Create 2.2.1.1 android hotfix patch

### ISSUE THREAD

**Issue by vikulin** [web3j/web3j#300]
Please merge into 2.2.1 android version following commits:
https://github.com/web3j/web3j/commit/86804b808d61f1730cb644e38b7eb3b949fb828a
and
https://github.com/web3j/web3j/commit/b44a07fcb246aefc5fd90fa63e688a7161f1ce7e

3.x.x is blocked by https://github.com/web3j/web3j/issues/299

**Comment by youkpan** [comment]
Thanks
